### PR TITLE
Fixes #12025 - use mkisofs instead xorriso for remastering

### PR DESCRIPTION
**Pull request by lzap**

### CODE REVIEW

**Comment by lzap** [comment]
Note: When merging this, add the script to packages and dependencies as well.

File: 25-minimize.ks [hunk 1]
```diff
@@ -65,6 +65,7 @@ echo " * locking root account"
 passwd -l root
 
 echo " * cleaning up yum cache and removing rpm database"
+rpm -qa > /usr/packages_list
```
Comment (lzap) [inline 25-minimize.ks]: I realized we don't carry list of installed packages anymore (we delete RPM database to save some space). This gives a list that is reported by foreman-debug tool.

**Comment by stbenjam** [comment]
> Note: When merging this, add the script to packages and dependencies as well.

@lzap Can you clarify what this means?

**Comment by lzap** [comment]
Sure, I will raise a PR in our packaging repo to distribute the script in the distribution packages. We haven't distributed it yet.

File: root/usr/lib64/ruby/vendor_ruby/discovery/screen/info.rb [hunk 2]
```diff
@@ -10,9 +10,13 @@ def screen_info action_proc, message, error_message, success_screen, failure_scr
...
+
+  # facter sometimes leaves messages on stdout, refresh before/after
+  Newt::Screen.refresh
+  result = action_proc.call
   Newt::Screen.refresh
 
-  if action_proc.call
+  if result
     success_screen
   else
```
Comment (lzap) [inline root/usr/lib64/ruby/vendor_ruby/discovery/screen/info.rb]: There is no good way to silence facter so this at least hides the output when it ocassionally print a message.

**Comment by stbenjam** [comment]
> Sure, I will raise a PR in our packaging repo to distribute the script in the distribution packages. We haven't distributed it yet.

So this is a note for you or...? Is this PR ready to be merged or do you need to do this first?

**Comment by lzap** [comment]
RTM, tested. Thanks!

**Comment by lzap** [comment]
Added `implantisomd5` as well for the media check.

**Comment by stbenjam** [comment]
Thanks, tested and remastering works fine with the new script

Merged as 9dbe1197f3c2379ea8f997a8cd5fc95c179a6fea

**Pull request closed without merging**